### ISSUE TITLE
Default setting for Zeppelin livy.sql interpreter causes RuntimeException

### ISSUE THREAD

**Issue by AbbyJoy** [panovvv/bigdata-docker-compose#6]
Ran into another issue when using Zeppelin 9 image. When using the livy interpreter for SQL (`%livy.sql`), there is a stactrace (`java.lang.RuntimeException: Fail to callRemoteFunction, because connection is broken`) that prints for any command used. This is a known bug for the zeppelin 9 which is already fixed for zeppelin 0.9.1 release (https://issues.apache.org/jira/browse/ZEPPELIN-5197). Meanwhile, before upgrading after that release, the work around is to change the `zeppelin.livy.concurrentSQL`  Livy interpreter setting to true.

**Comment by panovvv** [comment]
Yep, I was able to reproduce that. SQL code in question still executed after that though. Adding a %livy.sql paragraph to the test notebook too, just so that these kinds of issues crop up earlier rather than late.

**Comment by panovvv** [comment]
Update: 

> SQL code in question still executed after that though. 

It only worked that way the first time I tried that - still no idea how :) On consequent runs, I had to set the `zeppelin.livy.concurrentSQL` like you mentioned...
Anyway, I'll let this issue ripe probably - easier to wait for the next Zeppelin release then dig into the process of setting interpreter settings in docker image....
